Name: Move/Reply-All Button Swap
Version: 1.2.1-1
Author: Jason Robitaille
Description: Swaps around the "move to folder" menu option and the "reply to all" button  (MIT license)

--- .orig/usr/palm/applications/com.palm.app.email/app/controllers/message-assistant.js
+++ /usr/palm/applications/com.palm.app.email/app/controllers/message-assistant.js
@@ -1606,8 +1606,8 @@
 		this.cmdMenuModel = {
 				visible:true,
 				items: [
+					{label:$L('Move'),      icon:'file',      command:'move'},
 					{label:$L('Reply'),     icon:'reply',     command:'reply'},
-					{label:$L('Reply all'), icon:'reply-all', command:'replyAll'},
 					{label:$L('Forward'),   icon:'forward-email', command:'forward'},
 					{label:$L('Delete'),    icon:'delete',    command:'delete'}
 				]};
@@ -1620,7 +1620,7 @@
 				items: [
 					this.markUnreadMenuItem,
 					this.markSetFlagMenuItem,
-					{label:$L('Move to folder...'),   command:'move'}
+					{label:$L('Reply all'), command:'replyAll'}
 				]};
 		this.controller.setupWidget(Mojo.Menu.appMenu, undefined, this.appMenuModel);
 		
